source/Ship.h

```diff
@@ -589,13 +591,15 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	bool neverDisabled = false;
 	bool isCapturable = true;
 	bool isInvisible = false;
+	bool isPlayerFlagship = false;
```

Comment:
If we have this variable, I think it could be used in many places in AI.cpp (not in this PR, it's already big).

Reply:
Could be a future PR